site/_layouts/community.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+{% include head.html %}
+
+<body id="{{ page.id }}">
+<div class="container-fluid site-outer-container">
```

Comment:
What about a little indentation for the `div` below the `body`? 😃

Reply:
thanks!